examples/planning/open_loop_test.ipynb

```diff
@@ -229,37 +227,38 @@
     "plt.hist(angle_errors.mean(-1), bins=100, label=\"Angle Error Histogram\")\n",
     "plt.legend()\n",
     "plt.show()"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "# Qualitative evaluation"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "## Visualise results\n",
     "We can also visualise some images with predicted and annotated trajectories using L5Kit visualisation features.\n",
     "\n",
     "In this example, we draw 20 images from our dataset and we visualise predicted and annotated trajectories on top of them."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "for frame_number in range(0, len(eval_dataset), len(eval_dataset) // 20):\n",
     "    \n",
     "    data = eval_dataloader.dataset[frame_number]\n",
-    "\n",
     "    data_batch = default_collate([data])\n",
-    "    \n",
+    "    for key, val in data_batch.items():\n",
+    "        if isinstance(val, torch.Tensor):\n",
+    "            data_batch[key] = val.to(device)\n",
```

Comment:
isn't enough to do 
```python
data_batch = {k: v.to(device) for k, v in data_batch.items()}
```

here? everything should be tensor here as far as I remember

Reply:
Yes, that also works. I've changed it.